src/omdata.h

```diff
@@ -119,7 +111,14 @@ class oter_id
         size_t size() const;
 };
 
+/// @todo: Deprecate these operators
```

Comment:
I don't think you need both `///` and `@todo`. Presuming that's correct the latter is clearer